### DISCUSSION TITLE
The Nuxt documentation is not clear.

### DISCUSSION THREAD

**Discussion by 1057105012** [nuxt/nuxt#27382] [Questions]
Sometimes, I don't want to know how many available solutions I have, I just want to know what the best suggestion is, like error handling, you tell me there are this, that, all kinds of errors. When I finish reading, I still don't understand how to create a global error page to catch all exceptions, and what cost I have to pay

**Comment by TheAlexLichter** [comment]
Hey 👋🏻
Thanks for your feedback - this should be covered in https://nuxt.com/docs/getting-started/error-handling

Let me know if this isn't helpful / what info is missing ☺️
